### PR TITLE
doc: show inherited members. references #3842

### PR DESCRIPTION
**Pull request by Zen-CODE**

### CODE REVIEW

**Comment by akshayaurora** [comment]
@Zen-CODE :dancers:

**Comment by dessant** [comment]
In it's current form it makes the doc hard to parse. Would it be possible tho list inherited stuff at the end of the class docs and mark clearly from which base class each item comes from?

**Comment by Zen-CODE** [comment]
I don't think shpinx allows this kind of customization.

http://sphinx-doc.org/ext/autodoc.html

Search for 'inherit' and you'll see the options are pretty limited. From my
comment on the original ticket:

"Although I'm not a fan of doing this as it makes the docs much more
verbose, I can't deny that showing inherited methods is clearer, more
complete and avoids many possible issues. I've seen that with our trainees."

On Sun, Dec 27, 2015 at 6:43 PM, dessant <notifications@github.com> wrote:

> In it's current form it makes the doc hard to parse. Would it be possible
> tho list inherited stuff at the end of the class docs and mark clearly from
> which base class each item comes from?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kivy/kivy/pull/3870#issuecomment-167425487>.
>



-- 
===============
ZenKEY - Become one.
http://www.camiweb.com/zenkey

**Comment by dessant** [comment]
In that case I would vote against the change, @tshirtman was not a fan of it either. We'll see how the rest of the team feels after using the online docs.

**Comment by akshayaurora** [comment]
ok, seems I was too eager in merging this...It would be good to have inherited members but not in this form :/

**Comment by Zen-CODE** [comment]
Okay, I'll commit a reversal of this change with a link to this discussion for easy reversal following a proper discussion.

**Comment by Zen-CODE** [comment]
Reversed following proper discussion. 

https://github.com/kivy/kivy/commit/16e710f89c78755c1b754b3f52c95cb8b0d608a0
